### PR TITLE
docs: document sandbox status & improve docs

### LINKED ISSUE #115: Docs: Include other similar tools
I should endorse a growing ecosystem of tools that manipulate SBOMs. Lets list other projects with a similar goal to bomctl.

- https://github.com/CycloneDX/sbom-utility

### PR DESCRIPTION
**Pull request by idunbarh**
## Description

This PR updates the README to ...
- Reflect bomctl's status as a sandbox project of the OpenSSF
- Document the commands that are implemented
- Document similar projects

See the rendered markdown at https://github.com/bomctl/bomctl/tree/sandbox-status

Fixes #115 

## Type of change

Please delete options that are not relevant.

- [ ] Documentation Updates

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,7 @@
 !/.vscode/tasks.json
 !/.yamllint
 !/docs/**/*.md
+!/docs/images/**
```
Comment (jhoward-lm) [inline .gitignore:28]: I think this targets one or more nested _folders_. How would you feel about something like

```suggestion
!/docs/images/*.png
!/docs/images/*.svg
```

**Review by jhoward-lm — APPROVED** [review]

**Review by ashearin — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
+- [Try `bomctl` in our gitpod playground](https://gitpod.io/?autostart=true#https://github.com/bomctl/bomctl-playground)
 
 > [!NOTE]
 > This is an experimental project under active development. We'd love feedback on the concept, scope, and architecture!
 
 ## Features
 
 - Work with multiple SBOMs in tree structures (through external references)
-- Fetch and push SBOMs using HTTPS, OCI, and GIT protocols
+- Fetch and push SBOMs using multiple protocols
```
Comment (MFry) [inline README.md:19]: nit: multiple supported protocols

File: README.md [hunk 3]
```diff
@@ -1,28 +1,31 @@
...
 - Leverage a `.netrc` file to handle authentication
-- Manipulate SBOMs with commands like `diff`, `split`, and `redact`
 - Manage SBOMs using a persistent database cache
-- Interface with OpenSSF projects and services like [GUAC](https://guac.sh/) and [Sigstore](https://www.sigstore.dev/)
+- __FUTURE__ - Manipulate SBOMs with commands like `diff`, `split`, and `redact`
+- __FUTURE__ - Interface with OpenSSF projects and services like [GUAC](https://guac.sh/) and [Sigstore](https://www.sigstore.dev/)
```
Comment (MFry) [inline README.md:23]: Interface with other OpenSSF projects...

File: README.md [hunk 4]
```diff
@@ -69,51 +72,112 @@ Build using the `Makefile`
 
 ## Commands
 
-### Fetch (Implemented)
+`bomctl` stores SBOMs in a cache database, this allows more complex cross SBOM document operations to exist. This is a fundemental concept; files are not directly modified, the cache is.
 
-Ability to retrieve an SBOM via several protocols:
+`bomctl` uses the [protobom library](https://github.com/protobom/protobom) to store the SBOM component graph in a SBOM format agnostic manner. When you interact with the `bomctl` cache, you are interacting with the [protobom library](https://github.com/protobom/protobom).
```
Comment (MFry) [inline README.md:77]: Suggestion: ... in an SBOM agnostic format.

File: README.md [hunk 5]
```diff
@@ -69,51 +72,112 @@ Build using the `Makefile`
...
+  - [list](#list)
+- SBOMs are outputted out of the cache
+  - [export](#export)
 
-and from various locations:
+### Export
 
-- Local Filesystem
-- OCI
+Eexport stored SBOMs to either standard output or the file system.
```
Comment (MFry) [inline README.md:89]: Typo export

File: README.md [hunk 6]
```diff
@@ -69,51 +72,112 @@ Build using the `Makefile`
...
 
-This includes recursive loading of external references in an SBOM to other SBOMs and placing them into the persistent cache. If SBOMs are access controlled, a user's [.netrc](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html) file to authenticate.
+```shell
+bomctl export [flags] SBOM_ID...
 
-### Diff (Planned)
+Flags:
+  -e, --encoding string    output encoding [spdx: [json], cyclonedx: [json, xml]] (default "json")
+  -f, --format string      output format [spdx, spdx-2.3, cyclonedx, cyclonedx-1.0, cyclonedx-1.1, cyclonedx-1.2, cyclonedx-1.3, cyclonedx-1.4, cyclonedx-1.5] (default "cyclonedx")
```
Comment (MFry) [inline README.md:96]: nit: SBOM output format

File: README.md [hunk 7]
```diff
@@ -69,51 +72,112 @@ Build using the `Makefile`
...
+  -h, --help               help for export
+  -o, --output-file FILE   path to output file
+```
 
-TBD
+### Fetch
 
-### Lint (Planned)
+Ability to retrieve SBOM files from URLs over HTTPS, OCI, or Git.
```
Comment (MFry) [inline README.md:103]: ... over several protocols, including HTTPS, OCI, and git.

File: README.md [hunk 8]
```diff
@@ -69,51 +72,112 @@ Build using the `Makefile`
...
 
-### List (Planned)
+Flags:
+  -h, --help               help for fetch
+      --netrc              Use .netrc file for authentication to remote hosts
+  -o, --output-file FILE   Path to output file
+```
 
-TBD
+This includes recursive loading of external references in an SBOM to other SBOMs and placing them into the persistent cache. If SBOMs are access controlled, a user's [.netrc](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html) file to authenticate.
```
Comment (MFry) [inline README.md:114]: .netrc file can be used to authenticate.

File: README.md [hunk 9]
```diff
@@ -69,51 +72,112 @@ Build using the `Makefile`
...
+  - Creating diffs in machine readable and human readable formats
+- `Enrich`
+  - Use the [Transparency Exchange API](https://github.com/CycloneDX/transparency-exchange-api) to find enrich sbom components with additional data.
+  - Interface with [GUAC](https://guac.sh/)
+- `Merge`
+  - Merge fields from similar components in two or more SBOM documents
+  - Merge components and component dependencies from two or more SBOM documents
+  - Flatten multiple SBOM document component dependencies into a single SBOM document
+- `Redact`
+  - Redact fields by regular express or field name while keeping tracibility to the original document
```
Comment (MFry) [inline README.md:168]: **expressions** or field **names **

File: README.md [hunk 10]
```diff
@@ -69,51 +72,112 @@ Build using the `Makefile`
...
+- `Split`
+  - Split an SBOM dependency tree into multiple files based on purl type or component identifier
+- `Trim`
+  - Trim an SBOM dependency tree based on purl type or component identifier
+- __Your suggestions too!__
+
```
Comment (MFry) [inline README.md:174]: Mention previous architecture decisions and where they can be found and maybe link to a search for ADR issues that were closed

**Review by MFry — COMMENTED** [review]
Some small QoL changes

**Pull request merged**